wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -11,11 +11,11 @@ Having trouble getting your [beatmap](/wiki/Beatmaps) ranked? You've come to the
 
 The first and most important step towards getting your map ranked is to get people looking at it; people willing to [mod](/wiki/Modding) your map can be found in the #modreqs channel, the [modding queues forum](https://osu.ppy.sh/community/forums/60) or by asking them directly after checking their signature for modding policies. Remember that all mods are useful - any perspective on your map can be used to make it a better one. Consider every suggestion a modder makes, and if you turn any down, post a detailed response in the map thread making it clear why. It makes them a better modder and makes you think about how your map can improve! Also, if you found the mod particularly useful, be sure to give the post a [kudosu star](/wiki/Gl ossary).
 
-Your map's star priority is important when it comes to moving on to the next step. If a modder particularly likes your map they might give it a kudosu star. The more you work to make your map better, the more likely the modder will give you a kudosu star. You can also gain stars by modding other people's maps and spending the kudosu stars on your own mapset. If you don't think you're very good at modding, practice makes perfect! Small suggestions can really help another mapper out, so don't be afraid to give modding a go.
+Your map's hype train is important when it comes to moving on to the next step. If a modder likes your map, they might give it hype. Hype allows your mod to get noticed more by Beatmap Nominators.
```

Comment:
Calling the function as "hype train" would sound like it's only for popular maps. Infact, anyone can spend one hype point towards the map (They have 10 at a time, one per map and regenerates one per hour)

Reply:
This was fixed